mmcv/parallel/data_parallel.py

```diff
@@ -18,13 +18,25 @@ class MMDataParallel(DataParallel):
     Args:
         module (:class:`nn.Module`): Module to be encapsulated.
         device_ids (list[int]): Device IDS of modules to be scattered to.
-            Defaults to None when GPU is not available.
+            Use None when GPU is not available. Note this is different
+            from pytorch's DataParallel where None means use all gpu devices.
         output_device (str | int): Device ID for output. Defaults to None.
         dim (int): Dimension used to scatter the data. Defaults to 0.
     """
 
     def __init__(self, *args, dim=0, **kwargs):
-        super(MMDataParallel, self).__init__(*args, dim=dim, **kwargs)
+        device_ids = kwargs.get('device_ids', None)
```

Comment:
```python
        device_ids = kwargs.get('device_ids', None)
        if device_ids is None:
            device_ids = []
        if device_ids == []:
```
-> 
```python
        device_ids = kwargs.get('device_ids',  [])
        if len(device_ids) == 0:
```

Reply:
device_ids could be None (although not here)